operations/bills.md

```diff
@@ -102,12 +102,50 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
             },
             "OrderItems": [],
             "PaymentItems": [],
-            "AssigneeData": {
+            "OwnerData": {
                 "Discriminator": "BillCustomerData",
                 "Value": {
-                    "ItalianFiscalCode": null
+                    "Id": "26afba60-06c3-455b-92db-0e3983be0b1d",
+                    "Address": {
+                        "Line1": "Joe Doe street",
+                        "Line2": "Very long ave",
+                        "City": "Townston",
+                        "PostalCode": "154 00",
+                        "SubdivisionCode": "AU-NSW",
+                        "CountryCode": "AU"
+                    },
+                    "LegalIdentifiers": {
+                        "LegalIdentifier": "Value of legal identifier",
+                        "LegalIdentifier2": "Value of legal identifier 2",
+                    },
+                    "BillingCode": "billing code value",
+                    "LastName": "Last name value",
+                    "FirstName": "First name value",
+                    "SecondLastName": "Second last name value",
+                    "TitlePrefix": "Mistress",
+                    "FiscalIdentifier": "Fiscal identifier",
+                    "AdditionalTaxIdentifier": "Additional tax identifier"
```

Comment:
Can we put here more real data like in the Address please?

Reply:
I did what I could :) couldn't find more info on the other fields.